### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ThomasBarrial**

### CODE REVIEW

File: src/api/companies/controllers/users/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        role: role.toUpperCase(),
+      },
+      orderBy: {
+        createdAt,
+      },
+    });
+
+    if (!users) {
+      return res.status(404).json(errors.users[404]);
+    }
```
Comment (Karnak19) [inline src/api/companies/controllers/users/index.js:29]:
```js
if (!users) {
      res.status(404)
      throw new Error(errors.users[404]);
 }
```

So it goes to the Error Handling Middleware who will send the response

File: src/api/users/controllers/put.js [hunk 2]
```diff
@@ -23,6 +25,8 @@ const put = async (req, res, next) => {
     jobId,
   } = req.body;
 
+  if (password) hashedPassword = bcrypt.hashSync(password, 10);
+
```
Comment (Karnak19) [inline src/api/users/controllers/put.js:29]: should check if old password match before updating it

**Review by Karnak19 — CHANGES_REQUESTED** [review]
Nice work all around, just need to check thise 2 things

**Review by MatthiasWanner — COMMENTED** [review]
Have to modify put user as Karnak comment

**Review by Karnak19 — APPROVED** [review]

**Pull request merged**